src/OrbitGl/SelectionData.cpp

```diff
@@ -0,0 +1,43 @@
+
+// Copyright (c) 2023 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitGl/SelectionData.h"
+
+#include <utility>
+
+#include "ClientData/CallstackData.h"
+#include "ClientData/CallstackEvent.h"
+#include "ClientModel/SamplingDataPostProcessor.h"
+
+using orbit_client_data::CallstackData;
+using orbit_client_data::CallstackEvent;
+using orbit_client_data::CaptureData;
+using orbit_client_data::ModuleManager;
+using orbit_client_data::PostProcessedSamplingData;
+
+SelectionData::SelectionData(const ModuleManager& module_manager, const CaptureData& capture_data,
+                             PostProcessedSamplingData post_processed_sampling_data,
+                             const CallstackData* callstack_data)
+    : post_processed_sampling_data_(std::move(post_processed_sampling_data)) {
+  callstack_data_pointer_ = callstack_data;
```

Comment:
Nit: For consistency, you could move `callstack_data_pointer_` initialization up next to `post_processed_sampling_data_`.